src/store/LocalStore.ts

```diff
@@ -0,0 +1,20 @@
+import {create} from 'zustand';
+import {devtools} from 'zustand/middleware';
+import {shallow} from 'zustand/shallow';
+
+import createTriggersSlice, {TriggersSlice} from './TriggersSlice';
+
+/* Create store. For more information,
+check https://github.com/pmndrs/zustand/blob/main/docs/guides/typescript.md#basic-usage  */
+const useLocalStore = create<TriggersSlice>()(
```

Comment:
It is a global store, yet it should be local = without global space pollution. @haneabogdan, could you assist with that, based on your findings?